tools/spectral/ipa/package.json

```diff
@@ -18,14 +18,26 @@
   "license": "Apache-2.0",
   "author": "MongoDB",
   "main": "ipa-spectral.yaml",
+  "scripts" : {
+    "gen-ipa-changelog" : "npx auto-changelog"
```

Comment:
Follow up: f we introduce scripts consider moving other IPA related scripts from root later on (if makes sense)